backends/cpp_hart_gen/cpp/include/udb/bits.hpp

```diff
@@ -281,6 +310,21 @@ namespace udb {
     { std::remove_cvref_t<decltype(a.value())>::IsABits == true };
     { std::remove_cvref_t<decltype(a.value())>::PossiblyUnknown == false };
     { std::same_as<std::remove_cvref_t<decltype(a.unknown_mask())>, typename T::MaskType> };
+
+    // cast the bits to a known (not undefined type). Might raise if there are undefined bits
```

Comment:
```suggestion
    // cast the bits to a known (not undefined) type. Might raise if there are undefined bits
```
Also, what does "Might raise if there are undefined bits" mean?